modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-taglib/src/main/java/com/liferay/dynamic/data/mapping/form/taglib/servlet/taglib/DDMFormRendererTag.java

```diff
@@ -386,8 +396,17 @@ protected boolean isFormAvailable() {
 		DDMFormInstance ddmFormInstance = getDDMFormInstance();
 
 		if (ddmFormInstance != null) {
-			Group group = DDMFormTaglibUtil.getGroup(
-				ddmFormInstance.getGroupId());
+			Group group = null;
+
+			try {
+				group = GroupLocalServiceUtil.getGroup(
+					ddmFormInstance.getGroupId());
+			}
+			catch (PortalException portalException) {
+				if (_log.isDebugEnabled()) {
+					_log.debug(portalException);
+				}
+			}
```

Comment:
@slnn The tag is using _groupLocalService.fetchGroup(groupId);, why you change it to use getGroup()